cmd/swarm/main.go

```diff
@@ -248,6 +248,15 @@ Removes a path from the manifest
 Cleans database of corrupted entries.
 `,
 		},
+		{
+			Action:    importdb,
+			Name:      "importdb",
+			Usage:     "Imports chunks from one datastore to another",
+			ArgsUsage: "<source-datadir> s",
```

Comment:
no more <source-dir> <target-dir>? 
What's the target? How do I import from A to B? Would I use importdb <source-dir> --datadir=/path/to/target ?